src/rsa.rs

```diff
@@ -86,9 +92,29 @@ impl signature_impl::VerificationAlgorithmImpl for RSA_PKCS1 {
     }
 }
 
+macro_rules! rsa_pkcs1_padding {
+    ( $PADDING_ALGORITHM:ident, $digest_alg_name:expr,
+      $digest_alg:expr, $digestinfo_prefix:expr ) => {
+
+        #[doc="PKCS#1 1.5 padding with the "]
+        #[doc=$digest_alg_name]
+        #[doc=" digest algorithm."]
```

Comment:
Good point. `RSAKeyPair` is only available in the `use_heap` mode and `RSAPadding` is only useful for `RSAKeyPair.sign`. So, technically it doens't use the heap but there's no point in exposing it in the `no_heap` config. OTOH, no harm, no foul; I'm not sure it's worth extra effort to hide it either.

(BTW, "no_heap" got replace with "use_heap" yesterday.)